web/docs/handbook/contributing.mdx

```diff
@@ -12,10 +12,10 @@ Want to contribute? Why not jump into our GitHub repo and:
 
 - [Sponsor Supabase](https://github.com/sponsors/supabase).
 - [Answer Discussions](https://github.com/supabase/supabase/discussions).
-- Submit an issue.
+- [Submit an issue](https://github.com/supabase/supabase/issue).
 - Report a performance issue or a part of the documentation that you find confusing.
-- Create a pull request.
-- [Translate our Readme](https://github.com/supabase/supabase/issues/1341).
+- [Create a pull request](https://github.com/supabase/supabase/pulls).
```

Comment:
this one will also depend on which repo they're contributing to so maybe not required as a link just yet